backend/src/main/java/com/ambacam/model/Action.java

```diff
@@ -0,0 +1,55 @@
+package com.ambacam.model;
+
+import org.hibernate.validator.constraints.NotEmpty;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.io.Serializable;
+
+@Entity
+public class Action implements Serializable {
+
+	private static final long serialVersionUID = -2019398584857375323L;
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.AUTO)
+	private Long id;
+
+	@NotEmpty(message = "The name must not be empty")
+	@NotNull(message = "The name must not be null")
+	@Column(columnDefinition = "VARCHAR", nullable = false)
```

Comment:
We did not pay attention to this before. I prefer having the columnDefinition in lower case (`varchar` not `VARCHAR`).